### PR TITLE
Bump vibe.d + dependencies

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

**Comment by wilzbach** [comment]
image


Will fix dmd-beta and dmd-nightly later.

File: .travis.yml [hunk 1]
```diff
@@ -9,9 +9,14 @@ d:
  - ldc-beta
  - ldc
 
+addons:
+  apt:
+    packages:
+      - libevent-dev
```
Comment (ZombineDev) [inline .travis.yml]: I thought that `libevent-dev` is not needed anymore now that we're using `vibe-d:core` in configuration `vibe-core`?

Comment (wilzbach) [reply .travis.yml]: I thought so too, but I was hitting linking errors on Travis. I then concluded that getting the build to pass is the first priority.

Comment (ZombineDev) [reply .travis.yml]: I see, thanks for explaining.

**Pull request merged**